vms/docker/Dockerfile

```diff
@@ -19,5 +19,8 @@ RUN a2enmod headers expires ssl rewrite proxy proxy_fcgi authn_dbd authn_socache
 # Disable opcaching for dynamic PHP reloading
 RUN echo "opcache.enable=0" >> /etc/php/7.4/fpm/php.ini
 
+# Allow composer to download packages
```

Comment:
I don't think we want to be appending directly to the ini file, because in the setup instructions in the readme it's symlinked to the copy in the repo. So if you were to update it here, the repo would appear dirty because we've worked on a symlink.

Reply:
Also, we specifically disable allow_url_fopen in our current ini file for security reasons. While it's not a *huge* deal to enable it in a docker container, I don't think hot-altering the site's configuration just to get composer to work is the right direction. If you nee to alter it, it should happen in a docker-specific PHP config file and not the site's own config file.

Reply:
OK, I’m a little confused at this point. It seems like we don’t need the second commit at all, because `composer install` happens before the symlink that shuts off outside access. I’d been testing this by manually running it from the terminal, but of course at that point the access was locked down.

So what we end up as being my problem was just an out of date vendor folder. Which means that I still have no idea why everyone else was having Docker problems.

Reply:
Ah, but of course we still need php-curl and php-zip. So I’ll amend the PR to just be that and the safe update.